pkg/infra/tower.go

```diff
@@ -121,8 +119,6 @@ func (t *Tower) ResetTower() {
 }
 
 func (t *Tower) TotalAgents() int {
```

Comment:
This function is actually never used we could just remove it as well as `AgentsCount` in simEnv

Reply:
I don't have a problem with leaving it in - could be useful for logging later on. Happy to remove completely though.

Reply:
(so can we make the PR "remove unused functions" and remove both `AgentCount` and `TotalAgents` all together?)

Reply:
(sorry my GitHub doesn't show the new comment unless I refresh, feel free to do either)